src/agents/memory/session.py

```diff
@@ -102,268 +97,3 @@ async def pop_item(self) -> TResponseInputItem | None:
     async def clear_session(self) -> None:
         """Clear all items for this session."""
         ...
-
-
-class SQLiteSession(SessionABC):
```

Comment:
some people may import this class from agents.memory.session; for backward-compatibility, can you import of the class in this file? Adding a simple comment mentioning the necessity of backward compatibility would be appreciated too.